okta_openvpn.py

```diff
@@ -287,8 +287,8 @@ def __init__(self):
         self.config_file = None
         self.env = os.environ
         self.okta_config = {}
-        self.username_suffix = None
```

Comment:
The newer version of python supports only strings in configparser (and no None). So we change the types here to be strings.